examples/graphs/proteins_pytorch_geometric/adaptive.yaml

```diff
@@ -33,4 +33,4 @@ entrypoint: model_def:GraphConvTrial
 environment:
   image:
     cpu: determinedai/environments:py-3.7-pytorch-1.9-lightning-1.3-tf-2.4-cpu-b06dafb
-    gpu: determinedai/environments:cuda-11.1-pytorch-1.9-lightning-1.3-tf-2.4-gpu-b06dafb
+    cuda: determinedai/environments:cuda-11.1-pytorch-1.9-lightning-1.3-tf-2.4-gpu-b06dafb
```

Comment:
Why do some examples have cpu and cuda images, but some examples you removed the environment.image entirely?

Reply:
There was a number of PTL tests that had images explicitly specified but they in fact were the default images. This is the only experiment which is "pinned" (because of the reasons described in README).